feature-1/one-more-file.txt

```diff
@@ -0,0 +1 @@
+this file was added in feature-1 branch
```

Comment:
bla bla bla comment you've done a good work!!

Reply:
Thank you!!